Tutorials/CNTK_103A_MNIST_DataLoader.ipynb

```diff
@@ -21,7 +21,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 15,
```

Comment:
Gentle ping. Could you please rebase with master and reset the execution could. Appreciate your help.